cmd/celestia/internal/rpc/das.go

```diff
@@ -0,0 +1,36 @@
+package rpc
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/cmd/celestia/internal"
+)
+
+func init() {
+	DASCmd.AddCommand(samplingStatsCmd)
+
+	DASCmd.PersistentFlags().StringVar(
+		&internal.RequestURL,
+		"url",
+		"http://localhost:26658",
```

Comment:
repeated in lots of places, hence we should consider a config/defaults.go SOMEWHERE, where do you think would be best @vgonkivs?